social/src/main/java/bisq/social/userprofile/UserProfileService.java

```diff
@@ -197,4 +216,49 @@ public long getMinBurnAmount(Entitlement.Type type) {
             default -> 0;
         };
     }
+
+    private BaseHttpClient getBsqApiHttpClient() {
+        String userAgent = "Bisq 2";
+        //todo add providers to Bisq.config
+        List<String> providers = List.of(   // note the BSQ urls are different than the BTC urls
+                "https://bisq.mempool.emzy.de/bisq/api/",
+                "https://bisq.markets/bisq/api/",
+                "https://markets.bisq.services/bisq/api/");
+        String url = CollectionUtil.getRandomElement(providers);
+        Set<Transport.Type> supportedTransportTypes = networkService.getSupportedTransportTypes();
+        Transport.Type transportType;
+        if (supportedTransportTypes.contains(Transport.Type.CLEAR)) {
+            transportType = Transport.Type.CLEAR;
+        } else if (supportedTransportTypes.contains(Transport.Type.TOR)) {
+            transportType = Transport.Type.TOR;
+        } else {
+            throw new RuntimeException("I2P is not supported yet");
+        }
+        BaseHttpClient httpClient = networkService.getHttpClient(url, userAgent, transportType);
+        return httpClient;
+    }
+
+    private BaseHttpClient getBtcApiHttpClient() {
+        String userAgent = "Bisq 2";
+        //todo add providers to Bisq.config
+        List<String> providers = List.of(   // note the BSQ urls are different than the BTC urls
```

Comment:
We should move the providers to Bisq.config in application modules resource dir and add a inner static class Config to UserProfileService where we take those 2 provider lists. See DefaultApplicationService for other examples how its used. Typesafe is used for the config handling, will require prob. the dependency to get added to that module.